src/types/routerTypes.ts

```diff
@@ -18,12 +27,12 @@ export interface LoginParamList {
   changePin: AccountParams
   changeRecovery: ChangeRecoveryParams
   newAccountWelcome: {}
-  newAccountUsername: {}
-  newAccountPassword: {}
-  newAccountPin: {}
-  newAccountTos: {}
-  newAccountWait: WaitParams
-  newAccountReview: AccountParams
+  newAccountUsername: CreateFlowParams
+  newAccountPassword: CreateFlowParams
+  newAccountPin: CreateFlowParams
+  newAccountTos: CreateFlowParams
+  newAccountWait: WaitParams & CreateFlowParams
+  newAccountReview: AccountParams & CreateFlowParams
```

Comment:
Can we do `interface NewAcountReviewParams extends AccountParams , CreateFlowParams {}` inside the scene?